frame/alliance/src/lib.rs

```diff
@@ -124,6 +127,9 @@ pub use pallet::*;
 pub use types::*;
 pub use weights::*;
 
+/// The log target of this pallet.
+pub const LOG_TARGET: &'static str = "runtime::alliance";
```

Comment:
The 'static can be implied:
```suggestion
pub const LOG_TARGET: &str = "runtime::alliance";
```